In VMs we're super starved for entropy, and while we can and do wear a tin foil
hat, it's very hard to argue that rdrand and rdtsc add zero entropy.


--- linux-4.7/drivers/char/random.c~	2016-07-24 19:23:50.000000000 +0000
+++ linux-4.7/drivers/char/random.c	2016-07-29 19:08:18.406321150 +0000
@@ -1377,6 +1377,8 @@
 		if (!arch_get_random_seed_long(&rv) &&
 		    !arch_get_random_long(&rv))
 			rv = random_get_entropy();
+		else
+			credit_entropy_bits(r, 4); 
 		mix_pool_bytes(r, &rv, sizeof(rv));
 	}
 	mix_pool_bytes(r, utsname(), sizeof(*(utsname())));
